shared/message/platform/NUSenseData.proto

```diff
@@ -91,6 +91,17 @@ message IMU {
     uint32 temperature = 3;
 }
 
+message BackPanel {
+
+    // LEDs are not yet added in order to conserve bandwidth. However, they can be added later once the need arises.
+    message Buttons {
+        bool left   = 1;
+        bool middle = 2;
+    }
```

Comment:
Might be better if these were enums instead

Reply:
Instead of `bool`?

Reply:
If we only allow single presses yes. But now that I think about it, if we do it this way we can handle button combinations as well.

Reply:
It is also this way because it is like this in `RawSensors`, which has a boolean for each button.

Reply:
It is nested in case we need to add the LEDs, and it would be nice to have all of them nested in case there are other LEDs outside the backpanel.

Reply:
then do it as needed and add that in another pr that adds the LEDs or do it in here. Also, does the NUC really need to know about the LEDs?

Reply:
It doesn't, but the LEDs are in `RawSensors`.

Reply:
In that case, I think the LEDs are only worth sending through if we plan to log that information somewhere, maybe emit it to nusight so a tab can use it and it might be useful during a game when we are sshed into the robot. But then again a human can see the LEDs. I'll leave the decision to you but I'd prefer it if NUSense didn't have to encode messages that aren't important